include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -254,6 +254,10 @@ WARNING(cannot_assign_value_to_conditional_compilation_flag,none,
         "conditional compilation flags do not have values in Swift; they are "
         "either present or absent (rather than '%0')", (StringRef))
 
+WARNING(framework_search_path_includes_framework_extension,none,
+        "framework search path ends in \".framework\"; add directory containing "
+        "framework instead", (StringRef))
```

Comment:
The `(StringRef)` means the diagnostic takes one argument, but there's no `%0` in the format string. Since it's a path and probably long, I suggest just putting it at the end: `"…add directory containing framework instead: %0"`. It's a little unfortunate that that implies it's the path to use *instead,* but it's important to print the wrong path so that the developer can find out which argument was wrong, and I can't think of a better place to put it.

Reply:
That makes sense. I made the changes to the diagnostic.